roamance/src/components/social/comment/comment-section.tsx

```
@@ -0,0 +1,182 @@
+      )}
+
+      {/* Loading state */}
+      {isCommentsLoading ? (
+        <motion.div
+          initial={{ opacity: 0 }}
+          animate={{ opacity: 1 }}
+          className="flex flex-col items-center justify-center py-10"
+        >
+          <div className="relative">
+            <Loader2 className="h-7 w-7 text-ocean dark:text-ocean-light animate-spin mb-3" />
+            <div className="absolute inset-0 rounded-full animate-pulse-slow blur-xl bg-ocean/20 dark:bg-ocean/30 -z-10"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)